### PR TITLE
Added fuzzy search

### PR DESCRIPTION
**Pull request by ritz078**
screen shot 2016-06-24 at 10 02 36 pm

### CODE REVIEW

File: src/modules/ui/components/layout/index.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ import React from 'react';
 import VSplit from './vsplit';
 import HSplit from './hsplit';
 import SplitPane from '@kadira/react-split-pane';
+import FuzzySearch from 'react-fuzzy';
```
Comment (arunoda) [inline src/modules/ui/components/layout/index.js]: I like to move search component into a separate component and container by moving all these code out from the layout. Like I did for [shortcuts_help](https://github.com/kadirahq/storybook-ui/blob/master/src/modules/ui/containers/shortcuts_help.js).

And then simply use them inside the routing code. See: https://github.com/jtangelder/sass-loader

File: src/modules/ui/components/layout/index.js [hunk 2]
```diff
@@ -125,6 +208,9 @@ Layout.propTypes = {
   leftPanel: React.PropTypes.func.isRequired,
   preview: React.PropTypes.func.isRequired,
   downPanel: React.PropTypes.func.isRequired,
+  stories: React.PropTypes.arrayOf(React.PropTypes.object).isRequired,
+  showSearchBox: React.PropTypes.bool.isRequired,
+  actions: React.PropTypes.object.isRequired,
```
Comment (arunoda) [inline src/modules/ui/components/layout/index.js]: Don't pass the whole actions here. Just define the two actions, you really use.
It allows better documentation (self).

File: src/modules/ui/containers/layout.js [hunk 3]
```diff
@@ -3,9 +3,20 @@ import { useDeps, composeAll } from 'mantra-core';
...
+  const propShortCuts = pick(
+    shortcuts,
+    'showLeftPanel',
+    'showDownPanel',
+    'goFullScreen',
+    'showSearchBox'
+  );
+  return {
+    ...propShortCuts,
+    stories: api.stories,
```
Comment (arunoda) [inline src/modules/ui/containers/layout.js]: Do we need these two?

**Comment by arunoda** [comment]
@ritz078 This is great.
Once we okay with the above comments, we can proceed on this.

Love this.

**Comment by ritz078** [comment]
@arunoda done.

File: src/index.js [hunk 4]
```diff
@@ -27,7 +27,12 @@ export default function (domNode, provider) {
     ...uiModule.reducers,
   });
 
-  const reduxStore = createStore(reducer);
+  const reduxStore = createStore(reducer, (
```
Comment (arunoda) [inline src/index.js]: Could you submit this as another PR.

File: package.json [hunk 5]
```diff
@@ -46,16 +46,19 @@
   },
   "dependencies": {
     "@kadira/react-split-pane": "^1.4.0",
-    "redux": "^3.5.2",
-    "mantra-core": "^1.6.1",
     "babel-runtime": "^6.5.0",
+    "classnames": "^2.2.5",
```
Comment (arunoda) [inline package.json]: This one and the fuse.js is not directly used in this repo. Why don't you put this directly inside the `react-fuzzy` package.

**Comment by arunoda** [comment]
When I run the example, it's using some default font in the browser. Is it possible to use the fonts we used here: https://github.com/kadirahq/storybook-ui/blob/master/src/modules/ui/components/theme.js#L2
screen shot 2016-06-25 at 2 43 43 pm

**Comment by arunoda** [comment]
This is great. Thanks.
